release-notes/5.0/preview/api-diff/rc1/.Net/5.0-rc1_System.Buffers.md

```diff
@@ -0,0 +1,14 @@
+# System.Buffers
+
+``` diff
+ namespace System.Buffers {
```

Comment:
System.Buffers LGTM, provided we said the parameter rename was fine (source breaking change).

CC. @GrabYourPitchforks, @pgovind